thrift/from_wire_test.go

```diff
@@ -230,6 +230,27 @@ func TestValueFromWireSuccess(t *testing.T) {
 				"s": "foo",
 			},
 		},
+		{
+			// Enum with recognized value.
+			w: wire.NewValueI32(1),
+			spec: &compile.EnumSpec{
+				Name: "Op",
+				Items: []compile.EnumItem{{
+					Name:  "Add",
+					Value: 1,
+				}},
+			},
+			v: "Add",
```

Comment:
Added test for enum with multiple items that have the same value. (we currently just pick the first value).